utils/check-style/aspell-ignore/en/aspell-dict.txt

```diff
@@ -1734,6 +1734,7 @@ getMacro
 getOSKernelVersion
 getServerPort
 getSetting
+getSettingOrNull
```

Comment:
```suggestion
getSettingOrDefault
```

Reply:
Thank you @alexkats for the detailed review. I am working on the comments!

Reply:
Done, thanks!